index.js

```diff
@@ -55,10 +53,9 @@ db.run("CREATE TABLE IF NOT EXISTS requested_lectures (enrollment_id TEXT, lectu
 db.run("CREATE TABLE IF NOT EXISTS hits (date INTEGER, enrollment_id TEXT, user_agent TEXT)");
 db.run("CREATE TABLE IF NOT EXISTS token(id TEXT, description TEXT)");
 db.run("CREATE TABLE IF NOT EXISTS cache(id TEXT, value TEXT, expiration INTEGER)");
-//csvMigrator.migrate(db);
 db.close()
 
 //start server
-app.listen(app.get("port"), "127.0.0.1", function () {
-    console.log(`UniboClendar started on http://127.0.0.1:${app.get("port")}; press Ctrl-C to terminate.`);
+app.listen(app.get("port"), "0.0.0.0", () => {
```

Comment:
Actually, an environment variable with localhost as a fallback would be better than defaulting to ::/0